components/layout_thread/lib.rs

```diff
@@ -1019,18 +1020,20 @@ impl LayoutThread {
             }
             let display_list = (*rw_data.display_list.as_ref().unwrap()).clone();
 
+            debug!("Layout done!");
+
+            // TODO: Avoid the temporary conversion and build webrender sc/dl directly!
+            let wr_dl = rw_data.display_list.as_ref().unwrap().convert_to_webrender(self.id).finalize();
```

Comment:
I think it's be better to avoid obscure abbreviations here. Bytes are practically free! You can just write `webrender_display_list`.

Reply:
Bytes happen to be practically free but my screen can still only display 100 character in a line. :stuck_out_tongue:

Reply:
It's true that long names can lead to having to insert more new lines, but descriptive names make it a lot easier for people who are digesting the code for the first time.

Reply:
I still think it makes sense to just write webrender_display_list here.